dropgz/cmd/root.go

```diff
@@ -41,7 +41,7 @@ func init() {
 		os.Exit(1)
 	}()
 
-	// build root logger
+	// build root log
```

Comment:
looks like your find-and-replace was a little aggressive

Reply:
fixed

Reply:
I think if you go read the diff you'll agree this is not fixed, there are several cases of "logger" -> "log" changes which are not in the scope of this PR/don't make sense that shouldn't have been changed.

Reply:
fixed